### PR TITLE
Add the static read_fasttext method 

### PR DESCRIPTION
**Pull request by danieldk**
The other commit fixes a type signature and an unnecessary import.

### CODE REVIEW

**Comment by sebpuetz** [comment]
> The other commit fixes a type signature and an unnecessary import.

Have you checked whether the type signature actually ends up in Python? I experimented with that, too, but it seemed like it didn't actually get into the Python library.

File: Cargo.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ features = ["extension-module"]
 [dependencies]
 itertools = "0.8"
 failure = "0.1"
-finalfusion = { git = "https://github.com/finalfusion/finalfusion-rust.git", rev = "8e360c6" }
+finalfusion = { git = "https://github.com/finalfusion/finalfusion-rust.git", rev = "a7fec83" }
```
Comment (sebpuetz) [inline Cargo.toml]: Do we still need the git version, now that 0.8 is released?

**Review by sebpuetz — APPROVED** [review]

**Comment by twuebi** [comment]
> > The other commit fixes a type signature and an unnecessary import.
> 
> Have you checked whether the type signature actually ends up in Python? I experimented with that, too, but it seemed like it didn't actually get into the Python library.

https://github.com/danieldk/sticker-python/pull/2#discussion_r305341253

File: Cargo.toml (hunk 1 above)
Comment (danieldk) [reply Cargo.toml]: Nope, this can be updates to 0.8.

**Pull request merged**